hdmi/tinyaudio_hw.c

```diff
@@ -233,7 +235,16 @@ static int start_output_stream(struct stream_out *out)
         /*this will be updated once the hot plug intent
           sends these information.*/
         adev->card = DEFAULT_CARD;
-        adev->device = DEFAULT_DEVICE;
+	char value[PROPERTY_VALUE_MAX];
+	property_get("ro.hardware.audio",value,"0");
```

Comment:
- considering the property is read by vendor shared library, it is better to define a new property like ro.vendor.audio
- There should be space between , and value
- There should be space between , and "
- Indentation not in line with existing code